client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -46,8 +51,9 @@ export default function PlanOnlyThankYou( {
 
 	const [ resendStatus, setResendStatus ] = useState( RESEND_NOT_SENT );
 
+	const verifyEmailNoticeId = 'ecommerce-verify-email-notice';
```

Comment:
I decided to declare this constant closer to the block of code that makes use of it.